libs/wingsdk/test/simulator/simulator.test.ts

```diff
@@ -461,35 +526,35 @@ describe("in-place updates", () => {
     const myBucket2 = new Bucket(app2, "Bucket1");
     myBucket2.addObject("url.txt", myApi2.url);
 
-    // clear the state directory
-    fs.rmdirSync(stateDir, { recursive: true });
-
     const app2Dir = app2.synth();
     await sim.update(app2Dir);
 
     expect(updateTrace(sim)).toStrictEqual({
       added: [],
       deleted: [],
-      updated: ["root/Api1"],
+      updated: ["root/Api1"], // TODO: shouldn't Bucket also be listed here?
     });
 
     expect(simTraces(sim)).toEqual([
       "root/Api1 started",
       "root/Api1/Endpoint started",
+      "root/Api1/Policy started",
       "root/Bucket1 started",
+      "root/Bucket1/Policy started",
       "Update: 0 added, 1 updated, 0 deleted",
       "root/Api1/Endpoint stopped",
+      "root/Api1/Policy stopped",
+      "root/Bucket1/Policy stopped",
       "root/Bucket1 stopped",
       "root/Api1 stopped",
       "root/Api1 started",
       "root/Api1/Endpoint started",
+      "root/Api1/Policy started",
       "root/Bucket1 started",
+      "root/Bucket1/Policy started",
     ]);
 
-    const urlAfterUpdate = await (
-      sim.getResource("root/Bucket1") as IBucketClient
-    ).get("url.txt");
-    expect(urlAfterUpdate).not.toEqual(urlBeforeUpdate);
```

Comment:
Was this part of the test suddenly (checking that the contents of the bucket are unchanged) broken?

Reply:
I noticed this test was printing some errors when I ran vitest, and when I dug into it, it was because the test clears the state directory while the simulator is still running, causing errors when the resources save their state to a non-existent directory. I'm not sure that part of the test made sense, so I removed them.